client/src/components/Questions/ModalAddQuestion.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+import styled from 'styled-components';
+
+const StyledModal = styled.div`
+  text-align: center;
+`;
+
+const ModalAddQuestion = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  function toggleModal() {
+    setIsOpen(!isOpen);
+  }
+
+  return (
+    <div>
+      <StyledModal>
+        <button onClick={toggleModal}>Add Question</button>
+
+        <Modal
+          isOpen={isOpen}
+          onRequestClose={toggleModal}
+          contentLabel="My dialog"
+        >
+          <div>My modal dialog.</div>
+          <input type="button" value="close" onClick={toggleModal} />
+        </Modal>
+      </StyledModal>
+    </div>
+  );
+};
+
+export default ModalAddQuestion;
```

Comment:
Your tests are passing for this file, but in the browser, your modal is not popping up.
You said you'd be working on the modal more today--make sure to add more assertions to test whether or not the modal works.